### ISSUE TITLE
[CLI] Install error on a Windows Server 2019 WSL 2 

### ISSUE THREAD

**Issue by Anderson0abr** [teamdfir/sift#533]
Hi,

I'm trying to install sift-cli (server mode) on a Windows Server 2019 WSL 2 Ubuntu 20.04 and it always crashes at the same place. I saw a similar issue with the same error message (#528), but it doesn't seems to be the same problem. I'd appreciate a little help. Thanks in advance :)

![image](https://user-images.githubusercontent.com/3248826/122997057-fafc1b00-d381-11eb-89aa-8a52be87cbaa.png)

Here's my saltstack log:
[saltstack.log](https://github.com/teamdfir/sift/files/6697291/saltstack.log)

**Comment by ekristen** [comment]
It looks like your apt is broken. An unmet dependency cannot be installed. Looks like perhaps like ettercap-common failed.

Try running apt-get update && apt-get upgrade and see if that runs successfully. You can also try running `apt-get install --fix-broken`

**Comment by Anderson0abr** [comment]
I had already done this, but it didn't work.
Anyway, I tried again after some Windows updates and some new packages appeared to update on Linux. I don't know if Windows updates would influence that, but it worked after that.

TLDR: Update Windows; apt-get update && apt-get install --fix-broken && apt-get upgrade
